### PR TITLE
Clear up the README for the new user  

### PR DESCRIPTION
**Pull request by Ammar-Azman**
## Description

I am facing a problem of using the library which I don't understand what is the format required by the package
before I can pass it to the model. Hence, I added some note so it could be easier for other new user.

Checklist:
- [x] This PR has a meaningful title and a clear description.
- [x] The tests pass.
- [x] All linting tasks pass.
- [x] The notebooks are clean.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ series.head()
 | 3   | id_00     | 2000-01-04 | 232.704110 | 72       |
 | 4   | id_00     | 2000-01-05 | 317.510474 | 72       |
 
+> Note: The unique_id serves as an identifier for each distinct time series in your dataset. If you are using only single time series from your dataset, just create a single id number for this column.
```
Comment (jmoralez) [inline README.md:147]:
```suggestion
> Note: The unique_id serves as an identifier for each distinct time series in your dataset. If you are using only single time series from your dataset, set this column to a constant value.
```

**Review by jmoralez — CHANGES_REQUESTED** [review]
Thanks! Please also make this change in nbs/index.ipynb

**Comment by Ammar-Azman** [comment]
> Thanks! Please also make this change in nbs/index.ipynb

Okay done.

**Comment by jmoralez** [comment]
Thanks!

**Pull request merged**